client/components/PlanningList/index.jsx

```diff
@@ -31,7 +39,8 @@ class PlanningList extends React.Component {
                     agenda={currentAgenda}
                     onSpike={handlePlanningSpike}
                     onUnspike={handlePlanningUnspike}
-                    onClick={openPlanningEditor.bind(null, planning._id)}
+                    onClick={handleSingleAndDoubleClick.bind(null, this,
```

Comment:
A more common way to trigger an action from a component would be to add two props `onSingleClick` and `onDoubleClick`to `<PlanningItem/>`